src/util/RouterUtil.js

```diff
@@ -169,6 +169,10 @@ function (Okta, OAuth2Util, Util, Enums, BrowserFeatures, Errors, ErrorCodes) {
         router.navigate('signin/consent', {trigger: true});
       }
       return;
+    // We want the same view for FACTOR_REQUIRED & FACTOR_CHALLENGE
+    // In the new idx pipeline FACTOR_CHALLENGE API response does not contain a prev link
```

Comment:
is this going to be true? Or are the backend folks revisiting this? Asking to see if it makes sense to keep the comment or not

Reply:
Yeah, I think in the IDX world we are moving away from the pattern of making a call to prev link.